packages/client/src/__tests__/__snapshots__/noArgs.test.ts.snap

```diff
@@ -70,17 +70,24 @@ Invalid \`prisma.user()\` invocation:
 
 {
   data: {
+    id: 5,
+    name: 'Harshit',
+    email: 'a@a.de',
     likedArticles: {
-?     create?: ArticleCreateWithoutLikedByInput | ArticleCreateWithoutLikedByInput[] | ArticleUncheckedCreateWithoutLikedByInput | ArticleUncheckedCreateWithoutLikedByInput[],
-?     connectOrCreate?: ArticleCreateOrConnectWithoutLikedByInput | ArticleCreateOrConnectWithoutLikedByInput[],
-?     connect?: ArticleWhereUniqueInput | ArticleWhereUniqueInput[]
+      connect: null
+               ~~~~
```

Comment:
Errant red squiggly? 😅

Reply:
Ah, looking at other tests, is this part of the snapshotting?

Reply:
Yes, this is how we render errors. And github somehow has a highlighting for squigglies.